src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -242,3 +242,228 @@
     type: command
     short-summary: Update a virtual network peering.
 """
+
+helps['network'] = """
+    type: group
+    short-summary: Manages Network resources
+"""
+helps['network application-gateway'] = """
+    type: group
+    short-summary: Provides application-level routing and load balancing services 
+"""
+helps['network application-gateway address-pool'] = """
+    type: group
+    short-summary: Commands to manage app gateway address pools
+"""
+helps['network application-gateway frontend-ip'] = """
+    type: group
+    short-summary: Commands to manage app gateway front-end IP addresses
```

Comment:
>app [](start = 38, length = 3)

I would spell out application gateway since I've not officially seen it called app gateway anywhere.